src/erldns_txt.erl

```diff
@@ -77,15 +78,91 @@ split(Data, Parts) ->
 
 -ifdef(TEST).
 
+
+%% Known failure cases:
+%% \
+%% "
+%% "\"
 parse_test() ->
-  ?assertEqual(parse(""), []),
-  ?assertEqual(parse("test"), [[<<"test">>]]),
-  ?assertEqual(parse(lists:duplicate(270, "x")), [[list_to_binary(lists:duplicate(255, "x")), list_to_binary(lists:duplicate(15, "x"))]]),
-  ?assertEqual(parse(<<"test">>), [[<<"test">>]]),
-  ?assertEqual(parse("\"test\" \"test\""), [[<<"test">>], [<<"test">>]]),
-  ?assertEqual(parse("\\"), [[<<"\\">>]]),
-  ?assertEqual(parse("test\\;"), [[<<"test\\;">>]]),
-  ?assertEqual(parse("test\\"), [[<<"test\\">>]]).
+  ?assertEqual([], parse("")),
```

Comment:
EUNIT expects ?assertEqual(Expected, Value). These were breaking, and resulting in confusion on my part.